src/components/UI/Dialog/index.css

```diff
@@ -28,11 +28,7 @@
 .Dialog__element[open] {
   opacity: 1;
   transform: translateY(0);
-}
-
-.Dialog__element::backdrop {
-  /* var() doesn't work here */
-  background: rgba(13, 15, 28, 0.8);
+  box-shadow: 0px 0px 0px 100vmax var(--modal-backdrop);
```

Comment:
since modal backdrop can't use css variables, this is a workaround using a box-shadow that covers the whole screen